### PR TITLE
TWIS-35/Forgot-password

### PR DESCRIPTION
**Pull request by BaburDilshat**
# TWIS-35/Forgot-password

## Link to Story

https://re-boot.atlassian.net/browse/TWIS-35

## Description

Create a page for confirm reset password

## Screenshot(s) / GIF(s):
image

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] Dependency / Configuration update
- [ ] Documentation update

## Checklist:

- [x] My code follows the project's style guidelines
- [x] I have created a Storybook story for my component(s)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/app/_components/ForgotPassword/ForgotPassword.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        {/* 👇 Decorators in Storybook also accept a function. Replace <Story/> with Story() to enable it  */}
+        <Story />
+      </Box>
+    ),
+  ],
+} satisfies Meta<typeof ForgotPassword>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
```
Comment (Jack-Vu) [inline src/app/_components/ForgotPassword/ForgotPassword.stories.tsx:32]: you'll need to change the Story exports below to reflect what you are doing

File: src/app/_components/ForgotPassword/ForgotPassword.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import React from 'react'
+
+const ForgotPassword = (): JSX.Element => {
+  
+
+  return (
+    <Box display="flex" flexDirection="column">
+        <Typography mb={1}variant='h5'sx={{fontWeight:'bold'}}>Password reset</Typography>
+        <Typography mb={5} sx={{color: 'text.secondary'}} >Your password has been successfully reset. click confirm to set a new password</Typography>
+        <Button variant='contained' style={{backgroundColor:"#648DDB"}} id='confirm-reset-password'>Confirm</Button>
```
Comment (Jack-Vu) [inline src/app/_components/ForgotPassword/ForgotPassword.tsx:11]: you should add an interface and then a prop to handles the onClick for the confirm button something to hold the callback function

**Review by Jack-Vu — CHANGES_REQUESTED** [review]
Needs a couple of changes in your stories file and since your feature has a button you should also set up an onclick changeeventhandler for it

File: src/app/_components/ForgotPassword/ForgotPassword.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Box } from "@mui/material";
+import ForgotPassword from "./ForgotPassword";
+
+// More on how to set up stories at: https://storybook.js.org/docs/writing-stories#default-export
```
Comment (Muradil-Erkin) [inline src/app/_components/ForgotPassword/ForgotPassword.stories.tsx:6]: all comments in the stories can be removed at this point

File: src/app/_components/ForgotPassword/ForgotPassword.tsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import { Box, Typography, Button} from '@mui/material'
+import React from 'react'
+
+const ForgotPassword = (): JSX.Element => {
+  
+
```
Comment (Muradil-Erkin) [inline src/app/_components/ForgotPassword/ForgotPassword.tsx:6]: File should be formatted to have no extra whitespace / empty lines

**Review by Muradil-Erkin — CHANGES_REQUESTED** [review]

**Pull request closed without merging**